DoSomething/gladiator
# Send test email

#### What's this PR do?

Added a "test" button to the message sending page that will send a message to the email and name set as the contest `sender_email` and `sender_name`

![image](https://cloud.githubusercontent.com/assets/1700409/14503591/d27d77f8-017e-11e6-9418-3f472245e123.png)


#### How should this be manually tested?
Pull down branch. Set your email as the `sender_email` in a contest. Go to send an email. Click the "test" button and you should receive an email with the tokens replaced with your user info and the correct contest/competition/campaign information.

#### Any background context you want to provide?

This removes the previous need for `APP_DEBUG=true` on dev and QA environments because now we can just manually fire off a test instead of deciding who to send to based on which environment we are in.

#### What are the relevant tickets?
Fixes #198
